helm/cluster-cloud-director/templates/kubeadmcontrolplane.yaml

```
@@ -112,7 +115,9 @@ spec:
       {{- end }}
     postKubeadmCommands:  
       {{- if $.Values.network.staticRoutes }}
-      {{- include "staticRoutes" . | nindent 6}}
+      - systemctl daemon-reload
+      - systemctl enable static-routes.service
+      - systemctl start static-routes.service
```

```suggestion
      - systemctl enable --now static-routes.service
```

entirely up to you if you want to use it, it's just more succinct. (ref: https://documentation.suse.com/smart/linux/html/reference-systemctl-enable-disable-services/index.html#id-1.4)

---

Nice, thanks